app/src/main/java/com/zulip/android/database/DatabaseHelper.java

```diff
@@ -30,7 +30,7 @@
     private static final String DATABASE_NAME = "zulip-%s.db";
     // any time you make changes to your database objects, you may have to
     // increase the database version
-    private static final int DATABASE_VERSION = 8;
+    private static final int DATABASE_VERSION = 9;
```

Comment:
Database version is already updated here https://github.com/zulip/zulip-android/commit/b596202484f28639c22ec684f51572ff1767fda7